HAM/data_scientist/assets/algo/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# this base image works in both CPU and GPU enabled environments
+FROM substrafoundation/substra-tools:0.7.0
+
+RUN mkdir /.cache
+RUN chmod -R 777 /.cache
+
+RUN mkdir /model
+RUN chmod -R 777 /model
+
+# install dependencies
+RUN pip3 install pandas numpy sklearn pillow scipy 
+#tensorflow  pickle-mixin
+RUN pip3 install torch torchvision
+#RUN pip3 install keras==2.2.4
```

Comment:
It would be better to gather every dependencies in one requirements.txt file and to copy it in your Dockerfile